test_infra/scripts/cfn-nag.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+pushd ..
+
+# define stacks to validate
+stacks=( base lakeformation databases opensearch )
+
+# run cdk-nag against each stack
+for stack in "${stacks[@]}"
```

Comment:
Do we actually need to run this for each stack individually? Can't we just run a `cdk synth` and run cfn-nag on the entire output directory?

Also in case [this](https://gitlab.aws.dev/wwcs-proserve-etip-data-analytics/software-labs/sdlf/orion/-/blob/main/orion-commons/orion_commons/cicd/actions.py#L118) helps

Reply:
Yep good call out, let's do that.